frontend/src/views/AdminAllUsers.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+    <div>
+        <v-table>
+            <thead>
+                <tr>
+                    <th v-for="header in headers" :key="header.value">{{ header.text }}</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr v-for="user in testUsers" :key="user.ID">
+                    <td>{{ user.ID }}</td>
+                    <td>{{ user.Username }}</td>
+                    <td>{{ user.Displayname }}</td>
+                    <td>{{ user.Role }}</td>
+                </tr>
+            </tbody>
+        </v-table>
+    </div>
+</template>
+
+
+<script>
+//TODO: errors pop up when trying to import this, I think it's a backend issue?
```

Comment:
What error is popping up here? I'm trying this out and getting errors about axios not being able to be imported.

Reply:
Yes, those were the same ones I was seeing. I commented it out for now and put in the dummy data. It looks like your PR has the fixes to axios. 👍